test/0.8.25/vaults/contracts/Protocol__Deployment.t.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../../../contracts/0.8.9/EIP712StETH.sol";
+import "forge-std/Test.sol";
+
+import {CommonBase} from "forge-std/Base.sol";
+import {LidoLocator} from "../../../../contracts/0.8.9/LidoLocator.sol";
+import {StdCheats} from "forge-std/StdCheats.sol";
+import {StdUtils} from "forge-std/StdUtils.sol";
+import {Vm} from "forge-std/Vm.sol";
+import {console2} from "forge-std/console2.sol";
+
+interface ILido {
```

Comment:
Can't we use an already defined interface from `contracts/0.8.25/interfaces/..`?

Reply:
Unfortunately, no
contracts/0.8.25/interfaces/ILido does not contain: 
```
setMaxExternalRatioBP
initialize
resumeStaking
resume
setStakingLimit
```